Create ui-c8y-1023-82-8-unit-is-now-added-when-the-datapoint-unit-is-empty-and.md (#4757)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-82-8-unit-is-now-added-when-the-datapoint-unit-is-empty-and.md

---
date: ""

> title: Measurement's unit is now displayed when the data point has no unit defined

Reviewer: ```suggestion
title: Measurement's unit is now displayed even if the data point has no unit defined
```

> title: Unit is now added when the data point unit is empty and the measurement has a unit

Reviewer: ```suggestion
title: Measurement's unit is now displayed when the data point has no unit defined
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-66548
version: 1023.82.8
---

> When a new measurements is added, the data graph widget tooltip now properly displays the unit of measurement even when the data point itself does not have a unit defined. Previously, if a data point lacked a unit value, the measurement's unit would not be shown, making it unclear what unit the data was measured in. This change improves data clarity and helps you better understand the measurements in your dashboards without having to manually add units to each data point.

Reviewer: ```suggestion
When a new measurement is added, the "Data graph" widget tooltip now properly displays the measurement's unit, even when the data point itself does not have a unit defined. Previously, if a data point lacked a unit, the measurement's unit would not be shown, making it unclear which unit the data was measured in. This change improves data clarity and helps you better understand the measurements in your dashboards without having to manually add units to each data point.
```
